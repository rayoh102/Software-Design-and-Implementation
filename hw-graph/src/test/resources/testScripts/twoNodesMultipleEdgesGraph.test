# Tests adding two nodes, multiple edges to graph.

# Create a graph
CreateGraph twoNodesMultipleEdgesGraph

# Add two nodes
AddNode twoNodesMultipleEdgesGraph a
AddNode twoNodesMultipleEdgesGraph b

# Add five edges
AddEdge twoNodesMultipleEdgesGraph a a AA
AddEdge twoNodesMultipleEdgesGraph a b AB1
AddEdge twoNodesMultipleEdgesGraph a b AB2
AddEdge twoNodesMultipleEdgesGraph b a BA
AddEdge twoNodesMultipleEdgesGraph b b BB

# Print the nodes in the graph, the outgoing edges from a and the outgoing edges from b
ListNodes twoNodesMultipleEdgesGraph
ListChildren twoNodesMultipleEdgesGraph a
ListChildren twoNodesMultipleEdgesGraph b