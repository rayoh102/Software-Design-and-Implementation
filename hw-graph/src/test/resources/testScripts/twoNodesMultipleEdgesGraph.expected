# Tests adding two nodes, multiple edges to graph.

# Create a graph
created graph two_nodes_multiple_edges_graph

# Add two nodes
added node a to twoNodesMultipleEdgesGraph
added node b to twoNodesMultipleEdgesGraph

# Add five edges
added edge AA from a to a in twoNodesMultipleEdgesGraph
added edge AB1 from a to b in twtwoNodesMultipleEdgesGraph
added edge AB2 from a to b in twoNodesMultipleEdgesGraph
added edge BA from b to a in twoNodesMultipleEdgesGraph
added edge BB from b to b in twoNodesMultipleEdgesGraph


# Print the nodes in the graph, the outgoing edges from a and the outgoing edges from b
twoNodesMultipleEdgesGraph contains: a b
the children of a in twoNodesMultipleEdgesGraph are: a(AA) b(AB1) b(AB2)
the children of b in twoNodesMultipleEdgesGraph are: a(BA) b(BB)
